python/py-soundfile/Portfile

```diff
@@ -28,11 +28,11 @@ long_description    ${description} SoundFile can read and write sound\
                     PyPy 2.0+. SoundFile represents audio data as\
                     NumPy arrays.
 
-checksums           rmd160  81e6706d13bce43a40a3ed4bf436ead9c4af913f \
-                    sha256  9bc012512f16ae13e90a6d3458bb300097ef0990f05d7403ad90c691fce51988 \
-                    size    35000
+checksums           rmd160  2c6637646b56575bfd0831546ab8dc127d3bf11c \
+                    sha256  ca175b3131e9c41456cf0dc759a70177e3c83387b782b4632b7f4c097fbde846 \
+                    size    35008
```

Comment:
why did the checksums change? Since the version number remains the same this should likely be treated as a [stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates) - correct?

Reply:
Thanks, didn't know about those. Done